src/Compilers/CSharp/Test/Semantic/Semantics/AwaitExpressionTests.cs

```diff
@@ -70,15 +161,17 @@ public void DefaultAwaitExpressionInfo()
             Assert.Equal(0, info.GetHashCode());
         }
 
-        private AwaitExpressionInfo GetAwaitExpressionInfo(string text, out CSharpCompilation compilation, params DiagnosticDescription[] diagnostics)
+        private AwaitExpressionInfo GetAwaitExpressionInfo(string text,
```

Comment:
>GetAwaitExpressionInfo [](http://example.com/codeflow?start=36&length=22)

Consider reverting the changes to this method if not needed.

Reply:
I do